pkg/apiserver/registry/stats/multicastgroup/rest_test.go

```diff
@@ -91,20 +91,29 @@ func TestRESTList(t *testing.T) {
 		networkPolicyStatsEnabled bool
```

Comment:
I think we should remove this factor from tests too.

Reply:
If we remove this factor, we cannot test the result of MulticastGroup API is irrelevant to the NetworkPolicyStats feature because we have already asserted it. Not sure if this is good practice. Could you explain more?

Reply:
Then the condition would be kept forever? Could it be confusing in the future?

Reply:
updated